ledger/eval.go

```diff
@@ -480,6 +485,34 @@ func (eval *BlockEvaluator) transaction(txn transactions.SignedTxn, ad transacti
 		if !txn.Txn.Group.IsZero() && !eval.proto.SupportTxGroups {
 			return fmt.Errorf("transaction groups not supported")
 		}
+
+		if !txn.Lsig.Blank() {
```

Comment:
Hm, where are we checking that `LogicSig` has been enabled in the current protocol version? There's a comment somewhere about how `LogicSigVersion == 0` means no support, but I'm not sure where that's enforced.